example/burrow_integration/integration_eaxample.py

```diff
@@ -0,0 +1,171 @@
+    '''Bytecode was generated using remix editor  https://remix.ethereum.org/ from file iroha.sol. '''
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller=ADMIN_ACCOUNT_ID, input=bytecode)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    net.send_tx(tx)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    for status in net.tx_status_stream(tx):
+        print(status)
+    return hex_hash
+@trace        
+def get_engine_receipts(hash):
+    query = iroha.query('GetEngineReceipts', tx_hash=hash)
+    IrohaCrypto.sign_query(query, ADMIN_PRIVATE_KEY)
+    response = net.send_query(query)
+    print(response)
+    contract_add = response.engine_receipts_response.engine_receipts [0].contract_address
+    return contract_add
+
+@trace
+def add_asset(address) :
+    params = get_first_four_bytes_of_keccak(b'addAsset(string,string)')  
+    no_of_param=2
+    for x in range(no_of_param) :
+        params=params+left_padded_address_of_param(x,no_of_param)
+    params=params+argument_encoding('coin#domain') #asset id
+    params=params+argument_encoding('500') # amount of asset
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller=ADMIN_ACCOUNT_ID, callee=address, input=params)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response=net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+@trace
+def create_account(address) :
+    params = get_first_four_bytes_of_keccak(b'createAccount(string,string,string)') 
+    no_of_param=3
+    for x in range(no_of_param) :
+        params=params+left_padded_address_of_param(x,no_of_param)
+    params=params+argument_encoding('test') #source account id
+    params=params+argument_encoding('domain') #domain id
+    params=params+argument_encoding(test_private_key) #  key
```

Comment:
`createAccount` is using public key: https://iroha.readthedocs.io/en/main/develop/api/commands.html#create-account, so You can use:
```
test_public_key = IrohaCrypto.derive_public_key(private_key).decode('utf-8')
```
then:
```suggestion
    params=params+argument_encoding(test_public_key) #  key
```